### ISSUE TITLE
Installing pdm-bump causes pdm version regression

### ISSUE THREAD

**Issue by smokeythemonkey** [carstencodes/pdm-bump#3]
When using pdm-bump with pdm >2.0.0 I get this error

```
11:38:48 ❯ pdm bump pre-release -v
E2

PDM 1.15.5 is installed, while 2.0.3 is available.
Please run $ pipx upgrade pdm to upgrade.
Run $ pdm config check_update false to disable the check.
```

After upgrading via pipx I only get `E2` as the error

pdm-bump was installed with pipx `pipx inject pdm pdm-bump`

Not sure if this is a pdm or pdm-bump issue. Will close and move this to pdm if its not a pdm-bump issue.

**Comment by carstencodes** [comment]
First quick look tells me, that pdm bump uses the compatible version dependency declaration to pdm version 1.12.0. So pdm 2 is considered to be incompatible with this version.

My personal roadmap is to take care of pdm bump in August 2022 and create the next release within the next two weeks.

**Comment by carstencodes** [comment]
I'm far behind my schedule. On top, I've noticed that somehow the pep440 versionutils package conflicts with a newer revision of pdm. I'm working on it, but it might become the end of the month before the new version arrives

**Comment by carstencodes** [comment]
Fixed revision 0.5.0
